deploy/artifacts/manifests.yaml

```diff
@@ -3,9 +3,7 @@ kind: ClusterTask
 metadata:
   annotations:
     description: |
-      This is where a really long-form explanation of what is happening in
-      crane-apply ClusterTask would go.
-    migration.openshift.io/run-after: crane-transform
+      Generate YAML by applying transformations to the exported resources.
   labels:
     app: crane-runner
```

Comment:
The reconcile predicate labels needs to be present here so that these cluster tasks dont get filtered out right?

Reply:
Added labels from code for now